### ISSUE TITLE
Installation without docker

### ISSUE THREAD

**Issue by ugintl** [oceceli/last-automation#2]
If I do not want to use docker, then I can install it like a normal laravel app?

**Comment by oceceli** [comment]
Of course, you can. But make sure you have installed extensions for PHP
